libs/execution_context/ebpf_maps.c

```diff
@@ -939,10 +676,33 @@ ebpf_map_update_entry(
         return EBPF_INVALID_ARGUMENT;
     }
 
```

Comment:
Add a top level comment to this function describing what update does for helper/non-helper and how by design, UM API and helper functions can race with each other for writing into a value.

Reply:
Done